packages/genesys-spark-components/src/components/stable/gux-listbox/options/gux-option/gux-option.tsx

```diff
@@ -87,6 +87,7 @@ export class GuxOption {
       >
         <div class="gux-option-wrapper">
           <gux-truncate ref={el => (this.truncateElement = el)}>
+            <slot name="text" />
             <slot />
```

Comment:
Do both these slots serve the same purpose ? I know the default slot cannot be changed as it would be a breaking change ?

Reply:
yeah, I think they generally do but it keep the existing behaviour working as expected